docs/visualize.asciidoc

```diff
@@ -37,6 +37,7 @@ To create a visualization:
 * *Maps*
 [horizontal]
 <<tilemap,Coordinate map>>:: Associate the results of an aggregation with geographic
+<<regionmap,Region map>>:: Thematic maps where color intensity of a shape indicates corresponds to a metric's value
 locations.
 * *Time Series*
```

Comment:
Typo: "indicates corresponds". And "locations"?

Maybe, "Thematic maps where a shape's color intensity corresponds to a metric's value."

Also in the Coordinate map description, should it be "geographic data"? Missing a period at the end, too.